data/registry/tools-dotnet-purview-telemetry-sourcegenerator.yml

```diff
@@ -0,0 +1,40 @@
+title: Purview Telemetry Source Generator
```

Comment:
```suggestion
# cSpell:ignore Kieron Lanning
title: Purview Telemetry Source Generator
```

Reply:
This will make the spell checker  happy

Reply:
Reverted the .cspell.yml change, and applied your suggestion.